app.js

```diff
@@ -37,7 +37,11 @@ var auth = require('./app/auth/index'),
 if (httpsEnabled) {
      app = express().https({
         key: fs.readFileSync(settings.https.key),
-        cert: fs.readFileSync(settings.https.cert)
+        cert: fs.readFileSync(settings.https.cert),
+        honorCipherOrder: settings.https.honorCipherOrder || true,
```

Comment:
This shouldn't block old clients. The cipher list may cause problems but we have enough flexibility to deal with this with an update to the default yml settings.

I cant think when it is ever reasonable to let a client dictate the security of the connection.